swagger.yaml

```diff
@@ -4183,6 +4183,15 @@ definitions:
         type: string
       successful:
         type: boolean
+      average_rtt:
+        type: number
+        format: double
+        description: Average Round Trip Time in milliseconds.
+        x-go-name: "AverageRTT"
+      packet_loss_count:
```

Comment:
I'll change this to `packet_loss_percentage`, since it is how it is provided by ping. I've been pondering whether to keep it as a percentage or transform it to a counter, but I feel the conversion is mood because all values will be normalized anyway (percentage, counter). What we want is be able to determine a grade of packet loss, and a percentage or a counter will do for this.